napari-workshops/installation.md

```diff
@@ -156,33 +146,9 @@ as well and you can skip to the next section.
     ```
 
 ````{admonition} Errors launching?
-Sometimes, `napari` installation can fail on an M1 Mac due to mismatching
-dependencies on `pip`.
```

Comment:
We don't mix conda and pip anymore, so I think this stuff can be deleted -- I run exclusively on M1 macOS without issues.